tests/shell/shell_recorder/tutorial_wrapper/SyntaxCheck.md

```diff
@@ -0,0 +1,39 @@
+Test1
+```sh
+# Mountpoint:/test
+ # testcomment
+	kdb set /test/a a
+# RET:0
+    kdb set /test/b b
+# should yield 'a'
+# STDOUT:a
+kdb get /test/a
+kdb get /test/a
+a
```

Comment:
multiline outputs in the second form works fine (see Test 2 in SyntaxCheck.md)
for the first form (STDOUT:) newlines need to be replaced with the return character that has been introduced because of incompatiblities with os x (grep -z) 

but i can reverse it for the tutorial transpiler. for the actual shell recorder i want to keep the current syntax.